main.py

```diff
@@ -13,7 +13,7 @@
 )
 from lib.utils import PurgeCacheError, S3UploadError, purge_cache, upload_to_s3
 
-notify_celery = notify_celery.NotifyCelery()
+notifyCelery = notify_celery.NotifyCelery()
```

Comment:
This syntax is unusual for Python and our other codebases. Could we just do `from notify_celery import NotifyCelery` instead to avoid the name clash? 

(Alternatively, [this other comment](https://github.com/alphagov/notifications-govuk-alerts/pull/151#discussion_r683438638) may make it redundant.)

Reply:
Indeed it did - I'll resolve this one.